chromium_src/components/content_settings/browser/ui/cookie_controls_controller.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "components/content_settings/browser/ui/cookie_controls_controller.h"
+#include "content/public/browser/browser_context.h"
+
+#define IsOffTheRecord() IsOffTheRecord() || (true)
```

Comment:
Was chatting about this a bit with @mkarolin and it seems dangerous to override `IsOffTheRecord` this way, as it may cause unforseen problems in the future if upstream adds more calls to that function in here.

Can you instead do a function-style macro substitution of `OnCookieControlsIconStatusChanged` and just replace the last parameter with `false`?

See https://github.com/brave/brave-core/blob/master/chromium_src/services/network/restricted_cookie_manager.cc#L27-L33 for an example of a similar approach.

Reply:
Thanks for suggestion, I pushed the fix